src/AAVE/Modal/SupplyModal.jsx

```diff
@@ -249,6 +255,27 @@ function depositErc20(amount) {
     });
 }
 
+const maxValue =
+  symbol === "ETH" || symbol === "WETH"
+    ? Big(balance).minus(MIN_ETH_GAS_FEE).toFixed()
```

Comment:
If `balance < MIN_ETH_GAS_FEE`, the `maxValue` should be `0` rather than a negative number.